.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
     - GIT_SOURCE_REPO="https://github.com/git/git.git"
     - GIT_EARLIEST_SUPPORTED_VERSION="v2.0.0
     - GIT_LATEST_SOURCE_BRANCH="master"
+    - GIT_NEXT_SOURCE_BRANCH="next"
```

Comment:
You could also defined `GIT_LATEST_SOURCE_BRANCH="next"` for the time being.
That's why I called it `LATEST` and not `MASTER` back then 😄